### PR TITLE
Fix/feedbacks

### PR DESCRIPTION
**Pull request by thenav56**
- Use total_swipes instead of task_count.
- Remove total_user_groups from User Node.
- Set MAX_PAGINATION_LIMIT to 500
fixes https://github.com/mapswipe/mapswipe-meta/issues/73
fixes https://github.com/mapswipe/mapswipe-meta/issues/79
fixes https://github.com/mapswipe/mapswipe-meta/issues/81
fixes https://github.com/mapswipe/mapswipe-meta/issues/80

### CODE REVIEW

**Review by k9845 — APPROVED** [review]

File: manager-dashboard/app/views/Projects/ProjectDetails/index.tsx [hunk 1]
```diff
@@ -279,6 +306,12 @@ function ProjectDetails(props: Props) {
                         onChange={setShowFeaturedUpdateConfirmationTrue}
                         disabled={featuredUpdatePending || statusUpdatePending}
                     />
+                    {isPrivateProject && (
+                        <div className={styles.projectVisibility}>
```
Comment (frozenhelium) [inline manager-dashboard/app/views/Projects/ProjectDetails/index.tsx:310]: Add following styling for this
```css
.project-visibility {
    display: flex;
    align-items: center;
    gap: var(--spacing-small);
}
```

**Review by frozenhelium — APPROVED** [review]

File: community-dashboard/app/views/UserDashboard/index.tsx [hunk 2]
```diff
@@ -169,7 +168,7 @@ function UserDashboard(props: Props) {
     const totalSwipeTime = userStats?.userStats?.stats?.totalSwipeTime;
     const totalSwipeTimeLastMonth = userStats?.userStats?.statsLatest?.totalSwipeTime;
 
-    const totalUserGroup = userStats?.userStats?.stats?.totalUserGroups;
+    const totalUserGroup = userStats?.user?.userInUserGroups?.count ?? 0;
```
Comment (tnagorra) [inline community-dashboard/app/views/UserDashboard/index.tsx:171]: What is the difference between totalUserGroup and userGroupsLength?

File: example.env [hunk 3]
```diff
@@ -66,6 +66,7 @@ MANAGER_DASHBOARD_FIREBASE_MESSAGING_SENDER_ID=
 MANAGER_DASHBOARD_FIREBASE_APP_ID=
 MANAGER_DASHBOARD_SENTRY_DSN=
 MANAGER_DASHBOARD_SENTRY_TRACES_SAMPLE_RATE=
+MANAGER_DASHBOARD_COMMUNITY_DASHBOARD_URL=https://community-dashboard.mapswipe.org
```
Comment (tnagorra) [inline example.env:69]:
```suggestion
MANAGER_DASHBOARD_COMMUNITY_DASHBOARD_URL=
```

Should we have values on example.env?

**Review by tnagorra — CHANGES_REQUESTED** [review]

File: example.env (hunk 3 above)
Comment (thenav56) [reply example.env:69]: Yes.

File: community-dashboard/app/views/UserDashboard/index.tsx (hunk 2 above)
Comment (tnagorra) [reply community-dashboard/app/views/UserDashboard/index.tsx:171]: Fixed!

**Review by tnagorra — APPROVED** [review]

**Pull request merged**